tests/test_tti.py

```diff
@@ -61,7 +61,7 @@ def ricker_source(t, f0):
                                   time_order=2, space_order=space_order)
     rec, u1, _ = acoustic.forward(save=False)
 
-    tn = 50.0
+    tn = 100.0
```

Comment:
why?

Reply:
50 doesn't cover much more than the source start, I am not a big fan of running/benchmarking on zeros only. even 100 is small but bigger would mess up TRAVIS